runtime/doc/lua.txt

```diff
@@ -2871,6 +2814,72 @@ parents({start})                                            *vim.fs.parents()*
         (function) Iterator
 
 
+==============================================================================
+Lua module: highlight                                          *lua-highlight*
+
+
+Nvim includes a function for highlighting a selection on yank.
+
+To enable it, add the following to your `init.vim` : >vim
+    au TextYankPost * silent! lua vim.highlight.on_yank()
+<
+
+You can customize the highlight group and the duration of the highlight
+via: >vim
+    au TextYankPost * silent! lua vim.highlight.on_yank {higroup="IncSearch", timeout=150}
+<
+
+If you want to exclude visual selections from highlighting on yank, use: >vim
+    au TextYankPost * silent! lua vim.highlight.on_yank {on_visual=false}
+<
+
+See https://github.com/machakann/vim-highlightedyank for example
```

Comment:
I think we could drop this by now?

Reply:
Thank god, this is a pain to handle in doxygen.